Code_Exercises/Exercise_03_Vector_Add/solution.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * SYCL Quick Reference
+ * ~~~~~~~~~~~~~~~~~~~~
+ *
+ * // Include SYCL header
+ * #include <CL/sycl.hpp>
+ *
+ * // Default construct a queue
+ * auto q = sycl::queue{};
+ *
+ * // Allocate device memory
+ * auto * devPtr = malloc_device<T>(mycount, q);
+ *
+ * // Memcpy
+ * auto event1 = q.memcpy(dst, src, sizeof(T)*n, {dep_event1, dep_event2});
+ *
+ * // Free memory
+ * free(ptr);
+ *
+ * // parallel_for
+ * auto event2 = q.parallel_for(global_range, {event1, event2}, lambda);
+ *
+ * Make sure to call `wait` or explicitly pass dependent events
+ */
```

Comment:
Add a reminder to copy the memory back from device to host?

Reply:
I've included a comment further down about memcpying back to host so think this might be OK?